providers/defltprov.c

```diff
@@ -373,6 +373,7 @@ static const OSSL_ALGORITHM deflt_keyexch[] = {
     { "DH:dhKeyAgreement", "default=yes", dh_keyexch_functions },
 #endif
 #ifndef OPENSSL_NO_EC
+    { "ECDH:id-ecPublicKey", "default=yes", ecdh_keyexch_functions },
```

Comment:
The property may change if matt's PR goes in first..

Reply:
@mattcaswell the race is on! :P

Reply:
Ain't the first time, eh? :wink:

Reply:
Haha....well...I'm at "approval done"... :-)

Reply:
> Haha....well...I'm at "approval done"... :-)

Should I cheat and put it on hold to get this through first?? :stuck_out_tongue_closed_eyes:

Reply:
Actually. I wonder if it would be worth me holding back my PR for this one? There's a lot depending on this.